### PR TITLE
refactor(generative_ai): clean up hidden variables in code samples

### PR DESCRIPTION
**Pull request by msampathkumar**
## Description

Reduce hidden variables to maintain the simplicity and brevity of your code 
examples

### CODE REVIEW

File: generative_ai/gemini_tuning.py [hunk 1]
```diff
@@ -59,7 +65,7 @@ def gemini_tuning_advanced(project_id: str) -> sft.SupervisedTuningJob:
     # TODO(developer): Update and un-comment below lines
     # project_id = "PROJECT_ID"
```
Comment (gericdong) [inline generative_ai/gemini_tuning.py:66]: Should we consider change the comment? project_id is not valid anymore.

File: generative_ai/gemini_tuning.py [hunk 2]
```diff
@@ -84,22 +90,19 @@ def gemini_tuning_advanced(project_id: str) -> sft.SupervisedTuningJob:
...
-def get_tuning_job(
-    project_id: str, location: str, tuning_job_id: str
-) -> sft.SupervisedTuningJob:
+def get_tuning_job() -> sft.SupervisedTuningJob:
     # [START generativeaionvertexai_get_tuning_job]
     import vertexai
     from vertexai.preview.tuning import sft
 
-    # TODO(developer): Update and un-comment below lines
+    # TODO(developer): Update project_id, location
```
Comment (gericdong) [inline generative_ai/gemini_tuning.py:98]: Should we consider change the comment? project_id is not valid anymore.

File: generative_ai/gemini_tuning.py [hunk 3]
```diff
@@ -84,22 +90,19 @@ def gemini_tuning_advanced(project_id: str) -> sft.SupervisedTuningJob:
...
     # project_id = "PROJECT_ID"
```
Comment (gericdong) [inline generative_ai/gemini_tuning.py:99]: Should we consider change the comment? project_id is not valid anymore.

File: generative_ai/multimodal_embedding_image_video_text.py [hunk 4]
```diff
@@ -12,46 +12,40 @@
...
     # [START aiplatform_sdk_multimodal_embedding_image_video_text]
     import vertexai
 
     from vertexai.vision_models import Image, MultiModalEmbeddingModel, Video
+    from vertexai.vision_models import VideoSegmentConfig
 
-    # TODO(developer): Update values for project_id,
-    #            image_path, video_path, contextual_text, video_segment_config
-    vertexai.init(project=project_id, location="us-central1")
+    # TODO(developer): Update project_id
```
Comment (gericdong) [inline generative_ai/multimodal_embedding_image_video_text.py:33]: project_id is not valid anymore.

File: generative_ai/gemini_safety_config_example.py [hunk 5]
```diff
@@ -19,7 +19,13 @@ def generate_text(project_id: str) -> str:
...
+    from vertexai.generative_models import (
+        GenerativeModel,
+        GenerationConfig,
+        SafetySetting,
+        HarmCategory,
+        HarmBlockThreshold,
+    )
 
     # TODO(developer): Update and un-comment below line
     # project_id = "PROJECT_ID"
```
Comment (gericdong) [inline generative_ai/gemini_safety_config_example.py:31]: project_id is not valid anymore.

File: generative_ai/gemini_tuning.py [hunk 6]
```diff
@@ -28,7 +34,7 @@ def gemini_tuning_basic(project_id: str) -> sft.SupervisedTuningJob:
     # TODO(developer): Update and un-comment below lines
     # project_id = "PROJECT_ID"
```
Comment (gericdong) [inline generative_ai/gemini_tuning.py:35]: Should we consider change the comment? project_id is not valid anymore.

File: generative_ai/gemini_tuning.py (hunk 1 above)
Comment (msampathkumar) [reply generative_ai/gemini_tuning.py:66]: I agree

**Review by gericdong — APPROVED** [review]

**Pull request merged**